File: ProjectManagementAPI/ProjectManagementAPI/Entities/Project.cs

using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

namespace ProjectManagementAPI.Entities
{
    public class Project
    {
        public int ProjectId { get; set; }
        public string ProjectName { get; set; }
        public DateTime? StartDate { get; set; }
        public DateTime? EndDate { get; set; }
        public int Priority { get; set; }
        public int? ManagerId { get; set; }
    }
}

File: ProjectManagementWeb/ProjMgmtWeb/src/app/Model/Project.ts
import { User } from "./User";

export class Project{

    ProjectID:number;
    ProjectName:string;
    NumberOfTasks:number;
    CompletedTasks:number;
    IsCompleted:boolean;
    StartDate:Date;
    EndDate:Date;
    Priority:number;
    Manager:User;
}